src/react-imgix.js

```diff
@@ -116,16 +124,34 @@ export default class ReactImgix extends Component {
     };
   };
 
+  imgixParams = () => {
+    const params = {
+      ...defaultImgixParams,
+      ...this.props.imgixParams
+    };
+
+    const { faces, entropy } = params;
+
+    let crop = false;
+    if (faces) crop = "faces";
+    if (entropy) crop = "entropy";
+    if (params.crop) crop = params.crop;
+
+    let fit = false;
+    if (entropy || faces) fit = "crop";
+    if (params.fit) fit = params.fit;
+
+    return {
+      ...params,
+      crop,
+      fit
+    };
+  };
+
   render() {
     const {
-      auto,
       children,
       component,
-      customParams,
```

Comment:
`imgixParams` should be added to `this.props` here, no?

Reply:
I don't think so -- this is not setting the prop types for the component, this is destructuring props into local variables to make it easier to use in the `render` method. `imgixParams` is not used anywhere in render, only in `ReactImgix.buildSrcs()`